openpype/hosts/maya/api/lib.py

```diff
@@ -2571,7 +2572,8 @@ def _get_attrs(node):
             new_name = "{0}_baked".format(short_name)
             new_node = cmds.duplicate(node,
                                       name=new_name,
-                                      renameChildren=True)[0]
+                                      renameChildren=True,
+                                      inputConnections=copy_input_connections)[0]  # noqa
```

Comment:
```suggestion
                                      upstreamNodes=copy_input_connections)[0]  # noqa
```
Very confusing command flags in `cmds`, but for whatever reason `upsteamNodes` is the same as using the `Duplicate input graph` through the `Duplicate Special` UI menu.

Reply:
Duplicate input graph - basically means duplicate everything that's connected to the inputs (which is the upstream nodes). Even though they use 'synonyms' it's still pretty clear right? Or was there another reason you were getting confused?

Reply:
When using `Duplicate Special` with `duplicate input graph` enabled, you get this as mel commands;

```
buildEditMenu("MayaWindow|mainEditMenu");
DuplicateSpecialOptions;
performDuplicateSpecial true;
duplicateSpecialCallback OptionBoxWindow|formLayout247|tabLayout4|formLayout249|tabLayout5|columnLayout32 1; hideOptionBox;
duplicate -rr -un;
```

We want to duplicate the input graph to copy the image plane to the baked camera. Unless I've missed something about this PR?

Reply:
The input graph is easily way larger than the image plane. You'd be duplicating full rigs, with full animation curves, etc. Plus if you would just want to export inputs along then it's just as fine to duplicate just the input connections (without duplicating the actual source nodes) and allow Maya to export that along on maya scene export with `constructionHistory`,  `channels`, etc.

I suppose even when duplicating them, if you then end up exporting the camera as the selected entry then the image plane should NOT be part of the export since we're exporting without construction history in the extractor as far as I know.

The PR did discuss issues with the `matchmove` family not taking along an image plane on the export - totally fine issue. But what this does by duplicating upstream nodes should not be the way to solve that tiny issue.

Reply:
What is your solution for duplicating the camera with its image plane?

Can only see `upstreamNodes=True` working with the `cmds.duplicate` command.

Reply:
As far as I know we don't need to duplicate the image plane - we'd just need to ensure it gets exported along which it should if `inputConnections=True` for the duplicate (because then it is connected) and the maya export actually exports with inputs which I believe is `constructionHistory=True` flag on the file export.

I recall there being a way to duplicate just a selection of nodes and their connections in between them would be transferred to the duplicates but I'm not sure how to do that. Maybe it's just `cmds.duplicate([image_plane, camera])`. I know maya file export is capable of doing that even if you have input connections disabled.